modules/core/karrio/server/providers/extensions/ninja_van.py

```diff
@@ -0,0 +1,34 @@
+import django.db.models as models
+import karrio.server.providers.models as providers
+
+@providers.has_auth_cache
+class NinjaVanSettings(providers.Carrier):
+   class Meta:
+       db_table = "ninjaVan_settings"
+       verbose_name = "NinjaVan Settings"
+       verbose_name_plural = "NinjaVan Settings"
+
+   api_key = models.CharField(max_length=100, blank=True, null=True)
+   secret_key = models.CharField(max_length=100, blank=True, null=True)
+   track_api_key = models.CharField(max_length=100, blank=True, null=True)
+   track_secret_key = models.CharField(max_length=100, blank=True, null=True)
+   account_number = models.CharField(max_length=50, blank=True, null=True)
+   account_country_code = models.CharField(
+       max_length=3, blank=True, null=True, choices=providers.COUNTRIES
+   )
```

Comment:
You need to replace these the the client_id and client_secret that NinjaVan support